src/dnx/pal.h

```diff
@@ -1,7 +1,7 @@
 // Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
-void GetNativeBootstrapperDirectory(LPTSTR szPath);
+LPTSTR GetNativeBootstrapperDirectory();
```

Comment:
When I did the work initially, I was under the impression there was a desire to not use the stl string types in the first level boot strapper.  If it's okay to use them, I'm happy to do so.